src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +45,8 @@ public Movie add(Movie movie) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("from Movie ", Movie.class).getResultList();
+        }
```

Comment:
if you decided to write catch blocks for get() operations, let's add it here as well